modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,103 @@
+* ClusterTasks based on Tekton Catalog 0.14
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.1.
+
+=== Pipelines
+* Workspace support for VolumeClaimTemplates has been added. The VolumeClaimTemplate for a PipelineRun and TaskRun can now be added as a volume source for Workspaces. The tekton-controller then creates a `PersistentVolumeClaim` using the template that is seen as a `PersistentVolumeClaim` for all TaskRuns in the Pipeline. Thus you do not need to define the `PersistentVolumeClaim` configuration every time it binds a workspace that spans multiple tasks.
+* Support for improving audits:
+** The `PipelineRun.Status` field now contains the status of every `TaskRun` and the Pipeline specification used to instantiate a `PipelineRun` to monitor the progress of the `PipelineRun`.
+** Pipeline results are now added to the pipeline specification and `PipelineRun` status.
+** The `TaskRun.Status` field, now contains the exact Task specification used to instantiate the `TaskRun`.
+* Support to find the name of the PersistentVolumeClaim when a VolumeClaimTemplate is used as a volume source is now added using variable substitution.
+* Support to apply the default parameter to Conditions.
+* A TaskRun created by referencing a ClusterTask now adds the `tekton.dev/clusterTask` label instead of the `tekton.dev/task` label.
+* The `kubeconfigwriter` now adds the `ClientKeyData` and the `ClientCertificateData` configurations in the Resource structure to enable replacement of the pipeline resource type cluster with the kubeconfig-creator Task.
+* The feature-flags config map names and the defaults config map name are now customizable.
+* Support for HostNetwork in the PodTemplate used by TaskRun is now added.
+* An Affinity Assistant is now introduced to support node affinity in TaskRuns that share workspace volume. By default, this is disabled on OpenShift Pipelines.
+* The PodTemplate has been updated to specify `imagePullSecrets` to identify secrets that the container runtime should use to authorize container image pulls when starting a Pod.
+* Support for emitting warning events from the TaskRun controller if the controller fails to update the TaskRun.
+* Standard or recommended k8s labels are now added to all resources to identify resources belonging to an application or component.
+* The Entrypoint process is now notified for signals and they are propagated using a dedicated PID Group of the Entrypoint process.
+* The PodTemplate is now allowed to be set on a Task level at runtime using `TaskRunSpecs`
+* Support for emitting TaskRun lifecycle events such as the `taskrun started` and the `taskrun running` commands is added.
+* Support for using the `$context.<task|taskRun|pipeline|pipelineRun>.name` variable to reference the appropriate name when in PipelineRuns and TaskRuns.
+* The PipelineRun controller now emits a Kubernetes event every time that a Pipeline is started.
```

Comment:
@vdemeester isn't this similar to the item on line 37? Could it be combined into one?

Reply:
One is about `TaskRun`, the other about `PipelineRun` :sweat_smile: We could group them together.

Reply:
Done